### PR TITLE
make run-checks --static pass again w/shellcheck installed

### PR DESCRIPTION
**Pull request by chipaca**
We'd slipped, here.

### CODE REVIEW

File: packaging/arch/PKGBUILD [hunk 1]
```diff
@@ -36,13 +36,12 @@ prepare() {
...
 
   export GOPATH="$srcdir/go"
-  mkdir -p "$GOPATH"
 
   # Have snapd checkout appear in a place suitable for subsequent GOPATH. This
   # way we don't have to go get it again and it is exactly what the tag/hash
   # above describes.
-  mkdir -p "$(dirname "$GOPATH/src/${_gourl}")"
-  ln --no-target-directory -fs "$srcdir/$pkgname" "$GOPATH/src/${_gourl}"
+  mkdir -p "$(dirname "$srcdir/go/src/${_gourl}")"
```
Comment (mvo5) [inline packaging/arch/PKGBUILD]: Why is this change needed? What is shellcheck complaining about? It seems like this is a bit less readable now (or maybe that is just me)?

Comment (chipaca) [reply packaging/arch/PKGBUILD]: that one isn't shellcheck. It's right after it, inside the same guard as shellcheck, checking for uses of GOPATH that assume it's only ever got a single element in it.

I can also tell it to skip the PKGBUILD file.

Comment (mvo5) [reply packaging/arch/PKGBUILD]: Aha, I get it - its the ":" check. I think its fine unless @bboozzoo dislikes it.
